tutorials/getting-started/README.md

```diff
@@ -0,0 +1,8 @@
+# Getting Started on Chameleon Cloud Tutorials
```

Comment:
👍

Reply:
I saw later how you spool up jupyter right from a README.md file; we can do that here too.